python/paddle/fluid/dygraph/dygraph_to_static/program_translator.py

```diff
@@ -483,14 +500,24 @@ def func(x):
             return dygraph_func(*args, **kwargs)
 
         function_spec = FunctionSpec(dygraph_func, args, kwargs)
-        _, partial_program_layer = self._program_cache[function_spec]
+        concrete_program, partial_program_layer = self._program_cache[
+            function_spec]
 
         if args and isinstance(args[0], layers.Layer):
             # Synchronize self.training attribute.
             partial_program_layer.training = args[0].training
             args = args[1:]
-
-        return partial_program_layer(args)
+        try:
+            return partial_program_layer(args)
+
+        except BaseException as e:
+            # NOTE:
+            # 1. If e raised in compile time, e should have been attached ERROR_DATA before;
```

Comment:
Same as above comment about English grammar:
1. "e is raised", not the error raises itself. It is programmer/program raises it.
2. been attached to sth.

Reply:
Thanks, done